app/scripts/templates/complete_sign_in.mustache

```diff
@@ -0,0 +1,37 @@
+<div id="main-content" class="card">
+  {{#isLinkExpired}}
+  <header>
+    <h1 id="fxa-account-unlock-link-expired-header">{{#t}}Unlock link expired{{/t}}</h1>
```

Comment:
`account-unlock` => `verify-token` in all these IDs. Ugh, another inconsistency between noun and verb order, and again, I'm sure I did that. Double ugh.